### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by jdalton**
update deps

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
+.idea/
```
Comment (nirgit) [inline .gitignore:2]: already exists in line below, remove

File: package.json [hunk 2]
```diff
@@ -55,16 +55,18 @@
     "url": "https://github.com/nirgit/simple-xml-to-json/issues"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^24.1.0",
-    "eslint": "^8.40.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-jest": "^27.9.0",
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-replace": "^5.0.5",
```
Comment (nirgit) [inline package.json:59]: unused dependency, remove

**Review by nirgit — CHANGES_REQUESTED** [review]
cool. PR looks good @jdalton 👍 

two things before we merge this PR:

1. fix the build which now fails (`npm run build`)
2. see comment re unused dependency (dont forget to rebuild the package-lock after removal)

**Comment by jdalton** [comment]
Updated with issues fixed.

**Review by nirgit — APPROVED** [review]

**Comment by nirgit** [comment]
lgtm @jdalton

**Comment by jdalton** [comment]
Okay! @nirgit I lack merge rights, so I cannot merge.

**Comment by nirgit** [comment]
> Okay! @nirgit I lack merge rights, so I cannot merge.

sent you an invite, after you accept, i'll set up the relevant permissions (AFAIR)

**Pull request merged**